docs/source/tracking.rst

```diff
@@ -722,6 +720,10 @@ then make API requests to your remote tracking server.
         mlflow_set_experiment("/my-experiment")
         mlflow_log_param("a", "1")
 
+Note that to use the artifact storage capabilities of a remote server, the client need a direct access to the artifact store.
+The client is the one pushing directly artifacts to the artifact storage, and not through the tracking server.
+How to setup credentials for the artifact storage depends on the :ref:`Artifact Stores <artifact-stores>` technology setup for the server.
```

Comment:
```suggestion
How to setup credentials for the artifact storage depends on the :ref:`Artifact Stores <artifact-stores>` technology setup for the server. Same holds for the definition of write options, like giving other users (e.g. the Tracking Server UI) permissions to read the artifacts.  
```
I want to hint at the possible problem that the tracking server can't read the files written by the client, for example when the client pushed files to S3 without [EXTRA_ARGS](https://www.mlflow.org/docs/latest/tracking.html#amazon-s3)